libs/datasets/latest_values_dataset.py

```diff
@@ -83,6 +87,34 @@ def build_from_data_source(cls, source):
 
         return cls.from_source(source, fill_missing_state=source.FILL_MISSING_STATE_LEVEL_DATA)
 
+    @classmethod
+    def _aggregate_puerto_rico_data(cls, data):
+        _log.info("aggregating puerto rico")
+        pr_data = data[data[CommonFields.FIPS].isin(custom_aggregations.ALL_PR_FIPS)]
```

Comment:
I think instead of defining all of the fips codes manually, you can just do something like the following:
```
is_pr_county = data[CommonFields.FIPS].str.match("72[0-9][0-9][0-9]")
pr_data = pr_data.loc[is_pr_county]
```